Program.cs

```diff
@@ -12,20 +12,71 @@ public static List<string> ReadWordsFromFile(string filePath)
 
         foreach (string line in lines)
         {
-            string[] lineWords = line.Split(' ');
+            string[] lineWords = line.Split(' ')
+            .Select(word => CleanWord(word))
+            .ToArray();
             words.AddRange(lineWords);
         }
 
         return words;
     }
 
-    public static void Main()
+    private static string CleanWord(string word)
     {
-        List<string> wordsList = ReadWordsFromFile("C:\\Users\\Zoran\\Desktop\\Dario_Internship\\#3_Words\\text.txt");
+        string cleanedWord = new string(word.Where(c => !char.IsPunctuation(c)).ToArray());
```

Comment:
Interesting way to remove punctuation!
As a challenge, we could try to do it with `Regex.Replace`?

Reply:
Thanks,
I've found in a documentation a way to use Regax.Replace and have implemented it.